tools/phenomenal/nmr/metabolab.xml

```diff
@@ -0,0 +1,61 @@
+<tool id="metabolab" name="Metabolab">
+     <description>Processing vendor raw data for Multivariate analysis</description>
+     <requirements>
+       <container type="docker">docker-registry.phenomenal-h2020.eu/phnmnl/metabolab</container>
+     </requirements>
+     <command><![CDATA[     
+       echo $zipfile;
```

Comment:
I would suggest, for security reasons, to remove the `echo`, `ls` and `pwd` calls to avoid showing in the logs relevant internal directories. Thanks!